src/Symfony/Bridge/Monolog/Handler/MailerHandler.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Handler;
+
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Formatter\HtmlFormatter;
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\AbstractProcessingHandler;
+use Monolog\Logger;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+/**
+ * @author Alexander Borisov <boshurik@gmail.com>
+ */
+class MailerHandler extends AbstractProcessingHandler
+{
+    private $mailer;
+
+    private $messageTemplate;
+
+    /**
+     * @param callable|Email $messageTemplate
+     */
+    public function __construct(MailerInterface $mailer, $messageTemplate, int $level = Logger::DEBUG, bool $bubble = true)
+    {
+        parent::__construct($level, $bubble);
+
+        $this->mailer = $mailer;
+        $this->messageTemplate = $messageTemplate;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function handleBatch(array $records): void
```

Comment:
I've copied some methods from original abstract [MailHandler](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/MailHandler.php#L49) As it is not compatible with both 1.x and 2.x versions. Should I mention source of those methods somewhere?

Reply:
@BoShurik I do not know well other handlers
But if code copied from abstract, why not call them directly to avoid duplicated code? (if code is strictly identical)

any way, thank you for finishing it

PS: maybe you can friendly ping the previous reviewers so that they reviews again when available :)

Reply:
@noniagriconomie it has different signatures: [1.x](https://github.com/Seldaek/monolog/blob/1.x/src/Monolog/Handler/MailHandler.php#L46) and [2.x](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/MailHandler.php#L49). Any way to extend it and support both versions?

Reply:
ho i see :s